pkg/virtctl/vm/vm.go

```diff
@@ -134,6 +141,20 @@ func (o *Command) Run(cmd *cobra.Command, args []string) error {
 			return fmt.Errorf("Error stopping VirtualMachine %v", err)
 		}
 	case COMMAND_RESTART:
+		if gracePeriod != -1 && forceRestart == false {
+			return fmt.Errorf("Cannot set gracePeriod without --force=true")
```

Comment:
Can not set gracePeriod to `0` without `--force=true`

Reply:
corrected